components/raftstore-v2/src/raft/peer.rs

```diff
@@ -603,7 +603,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                 continue;
             }
             if let Some(instant) = self.peer_heartbeats.get(&p.get_id()) {
-                let elapsed = instant.saturating_duration_since(now);
+                let elapsed = now.saturating_duration_since(*instant);
```

Comment:
Good catch!
A minor comment: we can use instant.saturating_elapsed() to achieve the same purpose.  No need to have `now` variable explicitly.

Reply:
Yes, the current impl defines now in the out loop, maybe to avoid calculate now() for multiple times but I think it's not the bottleneck so just use saturating_elapsed should be ok here.